Tools/Slack/slack-unread.1s.py

```diff
@@ -1,209 +0,0 @@
-slack_call_command = ' href=slack:'
-
-channels_list_url = 'https://slack.com/api/channels.list'
```

Comment:
Guessing the Slack API has changed. Is there another up-to-date plugin that fills this need? If not, any chance of salvaging this?

Reply:
I tested this with an App-Level-Token but was getting `invalid_auth` after replacing the [deprecated methods](https://api.slack.com/changelog/2020-01-deprecating-antecedents-to-the-conversations-api), I will restore this as it may still be possible somehow to make it work.